front_end/entrypoints/main/MainImpl.ts

```diff
@@ -313,6 +314,12 @@ export class MainImpl {
         'timelineAsConsoleProfileResultPanel', 'View console.profile() results in the Performance panel for Node.js',
         true);
 
+    // JS Profiler
+    Root.Runtime.experiments.register(
+      'jsProfilerTemporarilyEnable', 'Enable JavaScript Profiler temporarily', /* unstable= */ false,
+      'https://developer.chrome.com/blog/js-profiler-deprecation/',
+      'https://bugs.chromium.org/p/chromium/issues/detail?id=1354548');
+
```

Comment:
This reverts a change made in 4f0b002c84bdb82cb092d9f374f3327e5adea67e - the experiment is now registered here, but (pre)enabled in `rn_inspector` using `RNExperiments.enableExperimentsByDefault`.